addons/web/static/src/views/basic_relational_model.js

```diff
@@ -788,6 +788,7 @@ export class StaticList extends DataPoint {
         this.records = [];
 
         this.handleField = params.handleField;
+        this.context = this.model.__bm__.localData[this.__bm_handle__].getContext();
```

Comment:
did you consider doing this in DataPoint directly? `this.context` is set there, but for an unknown reason, it isn't set to the value you expect.